os/arch/arm/src/armv7-m/up_hardfault.c

```diff
@@ -148,6 +155,7 @@ int up_hardfault(int irq, FAR void *context, FAR void *arg)
 
 		if (insn == INSN_SVC0) {
 			hfdbg("Forward SVCall\n");
+			lldbg("Forward SVCall\n");
```

Comment:
What is difference between hfdbg above and this lldbg?

Reply:
Added for testing, I have removed now.
Please check